src/main/java/com/tenx/ms/retail/order/domain/OrderEntity.java

```diff
@@ -0,0 +1,47 @@
+package com.tenx.ms.retail.order.domain;
+
+import com.tenx.ms.retail.order.rest.dto.OrderStatusEnum;
+import com.tenx.ms.retail.store.domain.StoreEntity;
+import lombok.Data;
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.List;
+
+@Entity
+@Data
+@Table(name = "purchase_order")
+public class OrderEntity {
+    @Id
+    @GeneratedValue
+    @Column(name="order_id", nullable = false)
+    private Long orderId;
+
+    @Column(name="status", nullable = false, length = 7)
+    private OrderStatusEnum status;
+
+    @Column(name = "order_date", columnDefinition = "datetime default current_timestamp",
+            insertable = false, nullable = false)
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date orderDate;
```

Comment:
@marielbysoares if you use java.time.LocalDateTime there is no need for workarounds with columnDefinition and temporaltype. not a blocker.